### PR TITLE
stream.js

### PR DESCRIPTION
**Pull request by dotnetCarpenter**
Submitting to jsDelivr as discussed in https://github.com/dionyziz/stream.js-website/pull/2#discussion_r48688471

### CODE REVIEW

File: files/stream.js/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "stream.js",
+  "repo": "dionyziz/stream.js",
+  "files": {
+    "include": ["stream.js", "stream.min.js", "stream.min.js.map"]
```
Comment (megawac) [inline files/stream.js/update.json]: I don't see the min or map files https://github.com/dionyziz/stream.js/tree/v1.0.1

Comment (jimaek) [inline files/stream.js/update.json]: There is no stream.min.js over here https://github.com/dionyziz/stream.js/tree/v1.0.1
The bot will fail

**Comment by dotnetCarpenter** [comment]
Sorry, hang on. I'm in the middle updating our two repos, https://github.com/dionyziz/stream.js and https://github.com/dionyziz/stream.js-website but got disturbed. Everything should be done by Monday the 18th of January.

**Comment by dotnetCarpenter** [comment]
@megawac I think I got everything on order already

File: files/stream.js/info.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author = "Dionysis Zindros"
+github = "https://github.com/dionyziz/stream.js"
+homepage = "http://www.streamjs.org/"
+description = "Lazy streams."
+mainfile = "stream.js"
```
Comment (jimaek) [inline files/stream.js/info.ini]: Use the `stream.min.js` here

File: files/stream.js/update.json (hunk 1 above)
Comment (dotnetCarpenter) [inline files/stream.js/update.json]: Should the unminified version `stream.js` be included here?

Comment (jimaek) [inline files/stream.js/update.json]: Its preferable that its not

**Comment by dotnetCarpenter** [comment]
@jimaek @megawac Should be ready now.

https://github.com/dionyziz/stream.js/tree/v1.1.0

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**